modules/vaos/app/controllers/vaos/v2/appointments_controller.rb

```diff
@@ -92,6 +99,11 @@ def appointments
           appointments_service.get_appointments(start_date, end_date, statuses, pagination_params, include_index_params)
       end
 
+      def eps_appointments
+        @eps_appointments ||=
+          eps_appointments_service.get_appointments(patient_id: Settings.vaos.eps.fake_patient_id)
```

Comment:
why `fake_patient_id`?

Reply:
To test out with local data, I'll change to patient_id